sparkmagic/example_config.json

```diff
@@ -1,20 +1,12 @@
 {
   "kernel_python_credentials" : {
-    "username": "",
-    "password": "",
-    "url": "http://localhost:8998",
-    "auth": "None"
+    "url": "http://localhost:8998"
```

Comment:
Would it be possible to keep the ability to specify password/username inline in the config for these credential settings?

Reply:
Yes I actually made these changes, I think I removed these before I did that and forgot to put them back in. I believe that this is handled in kernelmagics.py inside _initialize_auth() method. Theres also the _initialize_auth() method in remotesparkmagics.py to handle this for other inline magics. Again, not too sure I understand the difference though. They both have an identical module docstring (Runs Scala, PySpark and SQL statement through Spark using a REST endpoint in remote cluster. Provides the %spark magic.)